docs/api-reference/pipeline/reset.mdx

```diff
@@ -15,3 +15,10 @@ app.add("https://www.forbes.com/profile/elon-musk")
 # Reset the app
 app.reset()
 ```
+
+<Note>
```

Comment:
It should be a different file I believe similar to reset.mdx

Reply:
Cool. Added!